crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -602,14 +604,28 @@ where
     F: SystemParamFunction<Marker>,
 {
     func: F,
-    pub(crate) param_state: Option<<F::Param as SystemParam>::State>,
-    pub(crate) system_meta: SystemMeta,
-    world_id: Option<WorldId>,
+    state: Option<FunctionSystemState<Marker, F>>,
+    system_meta: SystemMeta,
     archetype_generation: ArchetypeGeneration,
     // NOTE: PhantomData<fn()-> T> gives this safe Send/Sync impls
     marker: PhantomData<fn() -> Marker>,
```

Comment:
Is this still needed with `Marker` now appearing in `FunctionSystemState<Marker, F>`?

Reply:
It no longer appears in `FunctionSystemState`, so I'm assuming it still needs to exist.